.github/workflows/main.yml

```diff
@@ -8,10 +8,10 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Set up Python 3.8
+    - name: Set up Python 3.7
```

Comment:
why not be explicit with `3.7.2`? according to [the docs](https://github.com/actions/setup-python#available-versions-of-python) it will use the latest bugfix version anyway, but we should pin it ourselves

Reply:
`3.7.2` doesn't appear to be a version they even support (why would they, no reason to support all bugfix versions). You can check all available versions [here](https://github.com/actions/python-versions/blob/master/versions-manifest.json).

Also why not use the latest bugfix version?

Reply:
because we know `3.7.2` works. Using the latest bugfix is probably fine then, but let's still specify it explicitly (3.7.7) instead of letting the action default to the latest